### PR TITLE
fixed #814 : cleaned up the viewer css

### LINKED ISSUE #814: Some css for viewer is applied only on product
Some of the css that defines the tools is applied on the product version only. 
i.e. Button sizes, Back to home button style... 
These changes should be applied in any instance of the viewer. We have to investigate which of them can be considered part of the theme and which are part of mapstore's viewer, separate and include them in the proper components or in the mapstore theme.

### PR DESCRIPTION
**Pull request by MV88**
fix #814 
- Moved into related css files the part of the viewer css 
- Moved some css code in the theme

### CODE REVIEW

File: web/client/components/misc/spinners/GlobalSpinner/css/GlobalSpinner.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#mapstore-globalspinner  {
+    margin: 0 !important;
```
Comment (offtherailz) [inline web/client/components/misc/spinners/GlobalSpinner/css/GlobalSpinner.css]: Are !important needed?

File: web/client/product/assets/css/viewer.css [hunk 2]
```diff
@@ -14,62 +14,19 @@ html, body, #container, .fill {
...
 }
-#page-manager #home-button {
-    float: right;
-}
-.hidden {
-    display: none !important;
-}
 
-.react-draggable .panel-heading {
+.react-draggable .draggable-header {
```
Comment (offtherailz) [inline web/client/product/assets/css/viewer.css]: This should be included in the theme

File: web/client/product/pages/Maps.jsx [hunk 3]
```diff
@@ -7,7 +7,7 @@
  */
 const React = require('react');
 
-require('../assets/css/viewer.css');
+// require('../assets/css/viewer.css');
```
Comment (offtherailz) [inline web/client/product/pages/Maps.jsx]: Why did you removed it, there are still some classes specific for the product viewer, isn't it?

Comment (MV88) [inline web/client/product/pages/Maps.jsx]: is not removed, is just the diff messing up.

File: web/client/product/assets/css/viewer.css (hunk 2 above)
Comment (MV88) [inline web/client/product/assets/css/viewer.css]: will do

File: web/client/components/misc/spinners/GlobalSpinner/css/GlobalSpinner.css (hunk 1 above)
Comment (MV88) [inline web/client/components/misc/spinners/GlobalSpinner/css/GlobalSpinner.css]: We will remove them as soon as we will have a new theme to switch

File: web/client/product/pages/Maps.jsx (hunk 3 above)
Comment (offtherailz) [inline web/client/product/pages/Maps.jsx]: My fault, this is not needed in Maps.jsx, you can remove it

**Pull request merged**